vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/SpaceManager.java

```diff
@@ -253,4 +254,14 @@ public void deleteSpaceById(String id) {
 
         return spaceLinkRepo.getLinkedFromSpaces(id);
     }
+    
+    @Override
+    public Iterable<Space> getAllSpacesWithIncomingLinkInfo(Iterable<Space> spaceList) {

```

Comment:
either change the name or don't give it a list of spaces, but right now you say "getAllSpaces" but also give it a list of spaces. What is it all spaces or just the passed ones?

Reply:
Changed the name. THe spaceList was a list of all spaces. Changed that too allSpaces as well.